notesolver/build.gradle.kts

```
@@ -0,0 +1,17 @@
+plugins {
+  id("org.openstreetmap.josm").version("0.8.0")
+}
+
+josm {
+  pluginName = "notesolver"
+  josmCompileVersion = "18360"
+  manifest {
```

```suggestion
josm.apply {
  pluginName = "notesolver"
  josmCompileVersion = "18360"
  manifest.apply {
```
The VSCode plugin seems to complain when this is written without `.apply`. Both variants are equivalent and when running Gradle from the command line both would work.